docs/user/functional_logs/index.md

```diff
@@ -23,6 +23,7 @@ This key is used to build a dictionary for the tree, which is carried by the `Tr
 This allows to serialize in an efficient way the tree (by not repeating the duplicated templates).
 
 When the children of a `ReportNode` is non-empty, the message of the corresponding `ReportNode` should summarize the children content.
+To that end, one or several values can be added after the `ReportNode` construction.
```

Comment:
Can you add a line explaining what happens when the message is read and a value is not set (the pattern is displayed and no exception is encountered)?

Reply:
In fact this behavior is up to the user's code, the pattern can be displayed, an exception thrown, or anything else. What you describe is what's done in the `print` overview method.